modules/standard/Random.chpl

```diff
@@ -141,15 +141,18 @@ module Random {
     compilerError("Random.fillRandom is only defined for numeric arrays");
   }
 
-  /* Shuffle the elements of an array into a random order.
+  /* Shuffle the elements of the array into a random order.
```

Comment:
I think `an` is actually better here (since it could refer to any array, not a specific one).  Or `a rectangular`.